test/e2e/conftest.py

```diff
@@ -291,54 +292,58 @@ def _gen_api():
 
 @pytest.fixture(scope='session')
 def parallel_data(workflow_api, stack_resources, testing_env_variables):
-    workflow_api = workflow_api()
-    parallel_data_s3_uri = 's3://' + \
-        stack_resources['DataplaneBucket'] + '/' + 'upload/' +\
-        testing_env_variables['SAMPLE_PARALLEL_DATA']
+    # Skip parallel data tests for regions where Amazon Translate
```

Comment:
These three lines are new. The rest of the lines were just shifted down. Not sure why github code diff is showing like a bunch of lines were removed and new ones added.